WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/blaze/BlazeCampaignsCard.kt

```diff
@@ -218,15 +218,24 @@ private fun CardDropDownMenu(moreMenuOptions: BlazeCampaignsCardModel.MoreMenuOp
         ) {
             DropdownMenuItem(
                 text = { Text(stringResource(id = R.string.blaze_campaigns_card_more_menu_view_all_campaigns)) },
-                onClick = { moreMenuOptions.viewAllCampaignsItemClick.click() }
+                onClick = {
+                    isExpanded = false
```

Comment:
@AjeshRPai - I did the same thing locally when testing the first PR. Still don't know why `onDismissRequest` is not being revoked. Thanks for fixing it.